app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -1349,14 +1349,19 @@ public void closePopup() {
 
     public void removePopupFromView() {
         if (windowManager != null) {
-            final boolean isCloseOverlayHasParent = closeOverlayBinding != null
-                    && closeOverlayBinding.closeButton.getParent() != null;
-            if (popupHasParent()) {
-                windowManager.removeView(binding.getRoot());
-            }
-            if (isCloseOverlayHasParent) {
-                windowManager.removeView(closeOverlayBinding.getRoot());
-            }
+            try {
+                if (popupHasParent()) {
+                    windowManager.removeView(binding.getRoot());
+                }
+            } catch (final IllegalArgumentException ignored) { }
```

Comment:
Maybe add that information on how to reproduce here. So if someone wants to tackle it, they don't have to search the PR on github

Reply:
I could not find a way to reproduce this part consistently. I fixed the error report above by replacing `closeOverlayBinding.closeButton.getParent() != null` with `closeOverlayBinding.getRoot().getParent() != null`. I made sure there were no crashes even without `try`-`catch`. Then I added `try`-`catch` just to make sure that even random crashes are not possible. But I don't know how to reproduce such random crashes.

Reply:
ahh i now see that this is the 4k line file. Then just add that the 2 try catches are trying to fix random crashes. At some point down the line someone is going to look at it and wonder why its there,

Reply:
Done